src/cljc/ataru/schema/form_schema.cljc

```diff
@@ -132,18 +132,19 @@
 (s/defschema FormWithContentAndTarjontaMetadata
   (merge FormWithContent {:tarjonta FormTarjontaMetadata}))
 
-(s/defschema Answer {:key                    s/Str,
-                     :value                  (s/cond-pre s/Str
-                                                         s/Int
-                                                         [s/Str])
-                     :fieldType              (apply s/enum ["textField"
-                                                            "textArea"
-                                                            "dropdown"
-                                                            "multipleChoice"
-                                                            "singleChoice"])
-                     (s/optional-key :label) (s/maybe (s/cond-pre
-                                                        LocalizedString
-                                                        s/Str))})
+(s/defschema Answer {:key                          s/Str,
+                     :value                        (s/cond-pre s/Str
+                                                               s/Int
+                                                               [s/Str])
+                     :fieldType                    (apply s/enum ["textField"
+                                                                  "textArea"
+                                                                  "dropdown"
+                                                                  "multipleChoice"
+                                                                  "singleChoice"])
+                     (s/optional-key :cannot-edit) s/Bool
```

Comment:
Is optional-key enough, doesn't this crash still because s/Bool is not inside s/maybe?